### PR TITLE
Remove community supported Spring Zeebe SDK references

### PR DESCRIPTION
**Pull request by akeller**
## Description

**Connectors is not part of this PR scope.**

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [ ] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/self-managed/connectors-deployment/connectors-configuration.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ You can configure the Connector runtime environment in the following ways:
 
 ## Connecting to Zeebe
 
+<!-- NEEDS ATTENTION -->
+
 In general, the Connector Runtime will respect all properties known to [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe).
```
Comment (akeller) [inline docs/self-managed/connectors-deployment/connectors-configuration.md:19]: @sbuettner Can we adjust this to move away from the community-supported project?

Comment (sbuettner) [reply docs/self-managed/connectors-deployment/connectors-configuration.md:19]: Yes, as soon as this has been done: https://github.com/camunda/connectors/issues/2896

**Comment by akeller** [comment]
@aleksander-dytko I've set this to `Blocked` on the DevEx board until Connectors delivers https://github.com/camunda/connectors/issues/2896.

**Comment by akeller** [comment]
@aleksander-dytko we have a blocker here with Connectors, but do you want me to split this PR and just include the non-Connectors references? How quickly do you want me to remove these?

File: docs/self-managed/connectors-deployment/connectors-configuration.md (hunk 1 above)
Comment (akeller) [reply docs/self-managed/connectors-deployment/connectors-configuration.md:19]: Leaving this for now, will follow up in a separate PR to remove it.

**Comment by akeller** [comment]
@aleksander-dytko I've removed Connectors from the scope of this PR, but do you have a recommendation for what I should do with the Tasklist API tutorial?

Do you want this PR to ship this week? If so, I will do the brute force method and delete remaining references and put at clean up issue into the DevEx backlog.

**Review by conceptualshark — APPROVED** [review]
There is one one SM reference to the community spring-zeebe project re: connectors as well. I'm not sure if you'd like the attention label on that, too, or to add it to the Connectors ticket so it doesn't get lost, but otherwise good to go from me. 👍

**Comment by akeller** [comment]
> There is one one SM reference to the community spring-zeebe project re: connectors as well. I'm not sure if you'd like the attention label on that, too, or to add it to the Connectors ticket so it doesn't get lost, but otherwise good to go from me. 👍

Can you add it to the Connectors ticket? @conceptualshark 

After a DM with @aleksander-dytko yesterday, I'm merging this.

**Pull request merged**